Form/DataTransformer/PhoneNumberToArrayTransformer.php

```diff
@@ -28,14 +28,24 @@ class PhoneNumberToArrayTransformer implements DataTransformerInterface
      */
     private $countryChoices;
 
+    /**
+     * @var int
+     */
+    private $format;
+
     /**
      * Constructor.
      *
      * @param array $countryChoices
+     * @param int $format
      */
-    public function __construct(array $countryChoices)
+    public function __construct(
+        array $countryChoices,
+        $format = PhoneNumberFormat::INTERNATIONAL
```

Comment:
Sorry, I wasn't clear before. The test cases were fine, they highlighted a breaking change. The problem is this line -
 the default option should be `NATIONAL` because that's what it was fixed to before.

Anything else is going to result in a change in behaviour for peoples' existing setups.

Reply:
Ok i revert test changes and set default format to national, thank you @rh389

Reply:
If you could add a few more tests to those cases to check the desired result with a couple of different formats, that'd be perfect. Thanks again!